AnnoyancesFilter/sections/popups.txt

```diff
@@ -99,6 +99,8 @@ sodapdf.com##.be-tooltip
 !
 indirimdemarket.com#$##social-media-modal { display: none!important; }
 indirimdemarket.com#$#body { overflow: visible!important; }
+indirimdemarket.com#$#.modal-backdrop { display: none !important; }
+~connect.deezer.com,sk7mobile.com,calkalasutasayrani.com,egyptianstreets.com,aipa551.com,aipa552.com,baba-mail.co.il,catwalkyourself.com,bkv.hu,tfc.tv,diziweb.net,blogkurdu.net,bitfeed.co,ports.com.ua,opensnow.com,watch5s.rs,deezer.com,pr-cy.ru,bitdefender.com##.modal-backdrop
```

Comment:
What about obsolete domains here? Have you checked it?

Reply:
Nope. I have no Idea how.

Reply:
1. Open the site from the list above (main page and some other pages are enough).
2. Search `##.modal-backdrop` via Filter log. If you don't find it - you can remove domain from the list.

Reply:
I can't find it in AdGuard for Window's Filtering Log. Like none of them.

Reply:
I will the extension for now.

Reply:
@zloyden Please check this site. `https://www.sk7mobile.com/main.do`.

<details>

![image](https://user-images.githubusercontent.com/76908410/109923571-e8d66e00-7cf9-11eb-8a29-518f940b265f.png)
</details>

Reply:
Something else is stopping me from scrolling.

Reply:
> I will the extension for now.

I forgot to say that you should use it instead of the app.

Reply:
> Something else is stopping me from scrolling.

```
sk7mobile.com#$#body { overflow: visible !important; }
sk7mobile.com#$#.modal-main { display: none !important; }
sk7mobile.com#$#.modal-backdrop { display: none !important; }
```

Reply:
For `sk7mobile.com`, you should remove the domain from the common list and add the rules above to the Annoyances - `popups.txt`.

Reply:
@zloyden Should I add it? I removed the obsolete ones.

Reply:
> Should I add it?

Yes, add them https://github.com/AdguardTeam/AdguardFilters/pull/76412#discussion_r587205072